src/status_im/chat/models/input.cljs

```diff
@@ -53,9 +53,11 @@
                 (let [{:keys [commands responses]} (get-in db [:contacts/contacts identity])]
                   (let [commands'  (mapv (fn [[k v]] [k [v :any]]) (merge global-commands commands))
                         responses' (mapv (fn [{:keys [message-id type]}]
-                                           [type [(get responses type) message-id]])
+                                           (when (get responses type)
+                                             [type [(get responses type) message-id]]))
                                          requests)]
                     (into commands' responses')))))
+         (remove nil?)
```

Comment:
Let's use `keep` instead of `map` with `(remove nil?)` https://clojuredocs.org/clojure.core/keep